### PR TITLE
adding Disk Detective job

### PR DESCRIPTION
**Pull request by mrniaboc**

### CODE REVIEW

**Comment by mrniaboc** [comment]
@brian-c Please let me know if you're not the right person to ask to merge changes to jobs.zoo anymore.

File: public/index.html [hunk 1]
```diff
@@ -84,8 +81,23 @@ <h3 class="other location-heading">Other</h3>
               <p>The OU are leading the Dissemination, Education, and Citizen Science work package of the EU Horizon2020 project 'Astronomy ESFRI and Research Infrastructure Clusters' (ASTERICS). A major activity of this work package is to create mass participation experiments and educational resources targeting the science-inclined public of college age and over, working with the Zooniverse team to build them. You will arrange international workshops to facilitate the design and delivery of the citizen science experiments, and act as the project manager for the experiments.</p>
             </div>
           </div>
+
+          <div class="job">
+            <p><a class="job-title" href="">Astrophysics Postdoc</a></p>
```
Comment (brian-c) [inline public/index.html]: Missing an `href` here?

**Comment by brian-c** [comment]
@mrniaboc I'm happy to merge things here, but I'm not notified of PRs automatically. Assign the PR to me or ping me on Slack. Sorry this got left behind.

**Comment by mrniaboc** [comment]
No worries. Thanks man.

File: public/index.html [hunk 2]
```diff
@@ -84,8 +81,23 @@ <h3 class="other location-heading">Other</h3>
...
+            <div class="job-description">
+              <p>This postdoc will perform research on citizen science developing new Zooniverse projects using the new project builder. The postdoc will
+work at NASA Goddard Space Flight Center with Marc Kuchner, PI of the
+<a href="http://www.diskdetective.org">Disk Detective</a> Zooniverse project.
+Project concepts involve protoplanetary disks, debris disks, AGN, and
+brown dwarfs, but experts in any field of astrophysics with can apply.
+Contact Marc.Kuchner@nasa.gov for more information.  </p>
```
Comment (brian-c) [inline public/index.html]: Wanna link that email address? `<a href="mailto: Marc.Kuchner@nasa.gov">Marc.Kuchner@nasa.gov</a>`

Comment (mrniaboc) [inline public/index.html]: Sure. Hold on and I'll update the PR

**Comment by mrniaboc** [comment]
There we go

**Pull request merged**